### PR TITLE
MUIC-585: [Android] Fix strings

### PR DESCRIPTION
**Pull request by deniss-mc**
**Jira issue:**
https://glia.atlassian.net/browse/MUIC-585

**Additional info:**
More fixes to the string after QA session

### CODE REVIEW

File: widgetssdk/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,14 +6,12 @@
...
-    <string name="glia_chat_upgraded_to_audio_call">Upgraded to Audio call</string>
-    <string name="glia_chat_upgraded_to_video_call">Upgraded to Video call</string>
+    <string name="glia_chat_upgraded_to_audio_call">Upgraded to audio call</string>
+    <string name="glia_chat_upgraded_to_video_call">Upgraded to video call</string>
     <string name="glia_chat_single_choice_card_hint">Tap on the answer above</string>
     <string name="glia_chat_not_started_hint">Send a message to start chatting</string>
-    <string name="glia_chat_file_download_success_message">File is downloaded successfully.</string>
-    <string name="glia_chat_file_download_fail_message">"File download is failed."</string>
-    <string name="glia_chat_file_open_file_title">"Open File"</string>
-    <string name="glia_chat_file_download_failed_msg">Unable to download the file at the moment, please try again later.</string>
```
Comment (dukhovnyi) [inline widgetssdk/src/main/res/values/strings.xml:16]: do we don't need these two strings anymore?  `glia_chat_file_download_failed_msg` looks in use in `ChatView.java`.

Comment (deniss-mc) [reply widgetssdk/src/main/res/values/strings.xml:16]: Those are duplicated - glia_chat_file_download_failed_msg is not used

Comment (deniss-mc) [reply widgetssdk/src/main/res/values/strings.xml:16]: glia_chat_file_download_failed_msg is used and is not removed - just moved lower

Comment (deniss-mc) [reply widgetssdk/src/main/res/values/strings.xml:16]: see line 14

**Review by dukhovnyi — APPROVED** [review]

**Review by jakob-mc — APPROVED** [review]

**Pull request merged**